package.json

```diff
@@ -7,21 +7,23 @@
     "test": "test"
   },
   "scripts": {
-    "bower": "bower install",
+    "bower": "./node_modules/.bin/bower install",
```

Comment:
There is no need for `./node_modules/.bin/` prefix as when scripts are executed it's added to PATH. so even if you have not installed bower globally, but have it locally it will work. so we should remove this prefix from all scripts